agent/local/state_test.go

```diff
@@ -829,10 +829,6 @@ func TestAgentAntiEntropy_Services_WithChecks(t *testing.T) {
 }
 
 var testRegisterRules = `
- node "" {
- 	policy = "write"
- }
-
```

Comment:
Not sure why this was here. There are only two tests that use these policy rules, and the node name is not empty in both of them.

Reply:
Likely a carry-over from aclv1 where `node` then does what `node_prefix` does now (there was no v1 "exact" match semantic in the rules language).